test/verify/check-storage-btrfs

```diff
@@ -0,0 +1,275 @@
+        b.wait_visible(self.card_row("Storage", location=mount_point))
+
+        # create subvolume
+        m.execute(f"btrfs subvolume create {subvol}")
+        b.wait_visible(self.card_row("Storage", name=os.path.basename(subvol)))
+
+        # devices overview
+        self.click_card_row("Storage", name=label)
+        b.wait_visible(self.card_row("btrfs volume", name=disk1))
+        b.wait_visible(self.card_row("btrfs volume", name=disk2))
+
+        # Allocate some space to see if the space is spread evenly
+        # m.execute(f"fallocate -l 50M {mount_point}/bigfile")
+
+    def testDefaultSubvolume(self):
+        m = self.machine
+        b = self.browser
+
+        disk1 = self.add_ram_disk(size=140)
+        label = "test_subvol"
+        mount_point = "/run/butter"
+        subvol = "cake"
+        subvol_path = f"{mount_point}/{subvol}"
+
+        m.execute(f"mkfs.btrfs -L {label} {disk1}")
+        self.login_and_go("/storage")
+
+        # creation of btrfs partition can take a while on TF.
+        with b.wait_timeout(60):
+            b.wait_visible(self.card_row("Storage", name=label))
+
+        b.wait_in_text(self.card_row("Storage", name=os.path.basename(disk1)), "btrfs device")
+
+        # Create a new btrfs subvolume and set it as default and mount it.
+        m.execute(f"""
+            mkdir -p {mount_point}
+            mount {disk1} {mount_point}
+            btrfs subvolume create {subvol_path}
+            btrfs subvolume set-default {subvol_path}
+            umount {mount_point}
+            mount {disk1} {mount_point}
+        """)
+
+        # Show a warning for mismounting in details.
+        b.wait_visible(self.card_row("Storage", name=subvol))
+        b.wait_visible(self.card_row("Storage", name=subvol) + ' .ct-icon-exclamation-triangle')
+        self.click_card_row("Storage", name=subvol)
+        b.wait_text(self.card_desc("btrfs subvolume", "Name"), subvol)
+
+        # Mount automatically on /run/butter on boot
+        b.click(self.card_button("btrfs subvolume", f"Mount automatically on {mount_point} on boot"))
+        b.wait_not_present(self.card_button("btrfs subvolume", f"Mount automatically on {mount_point} on boot"))
+
+        # No warnings on main page for either subvolumes
+        b.go("#/")
+        b.wait_visible(self.card("Storage"))
+        b.wait_not_present(self.card_row("Storage", name=subvol) + ' .ct-icon-exclamation-triangle')
+        b.wait_not_present(self.card_row("Storage", name="/") + ' .ct-icon-exclamation-triangle')
+
+    def testMountingHelp(self):
+        m = self.machine
+        b = self.browser
+
+        self.login_and_go("/storage")
+
+        disk = self.add_ram_disk(size=128)
+        uuid = "a32d61d6-9d75-4327-9bb9-3fa3627300ae"
+        mount_point = "/run/butter"
+
+        m.execute(f"mkfs.btrfs -U {uuid} {disk}")
+
+        # creation of btrfs partition can take a while on TF.
+        with b.wait_timeout(60):
```

Comment:
This can likely be reduced by now, the tests failed for a totally different reason (missing udisks2-btrfs...